R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
If the user supplies only one midnight observation, then technically the user has only supplied 23 hours worth of data, correct? They will be missing the last timestep (23:00 to 24:00) worth of information? 

I think I'd prefer this is technically "correct" than assume the user is habitually missing data.